README.md

```diff
@@ -12,7 +12,7 @@ The Wikipedia Library Card Platform app, available at http://wikipedialibrary.wm
 
 ## Documentation
 
-Further project documentation can be found in the [Wiki](https://github.com/WikipediaLibrary/TWLight/wiki). Our issue tracking takes place on the [Library-Card-Platform Phabricator](https://phabricator.wikimedia.org/project/board/2765/) board.
+Further project documentation will be available in the [Wiki](https://github.com/WikipediaLibrary/TWLight/wiki). Our issue tracking takes place on the [Library-Card-Platform Phabricator](https://phabricator.wikimedia.org/project/board/2765/) board.
```

Comment:
I think this is more confusing than the previous text. The documentation is already in the Wiki, but the new text implies that we haven't written it yet.

Reply:
No sir, I just made few edits no major line deletion. But the changes I have made are grammatically correct so Please consider to merge it. Thank you, sir.

Reply:
I think your other change is good, but this one is less accurate. "is available" would be preferable. "will be" implies a future state.

Reply:
Ok, sir, I will edit the required part.

Reply:
Please check now